app/gui2/package.json

```diff
@@ -62,6 +62,7 @@
     "semver": "^7.5.4",
     "sucrase": "^3.34.0",
     "vue": "^3.3.4",
+    "vue-react-wrapper": "^0.3.1",
```

Comment:
i think this should be a `devDependency` (`npm i -D vue-react-wrapper`) right? since it's only used during development (histoire)